libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -301,7 +300,7 @@ public boolean isRecenterButtonVisible() {
   }
 
```

Comment:
`takeScreenshot` with no parameters is still an open question 👇 https://github.com/mapbox/mapbox-navigation-android/pull/3436#pullrequestreview-467912684 because as a client does nothing i.e. you can't get the screenshot generated

> Also we should check `NavigationView#takeScreenshot` that doesn't require arguments and it's public too cc @JunDai

Reply:
Right. I don't want to dive into rearchitecting things. Let's follow up in a separate PR if needed.